tools/restatectl/src/commands/log/list_logs.rs

```diff
@@ -34,10 +36,11 @@ pub async fn list_logs(connection: &ConnectionInfo, _opts: &ListLogsOpts) -> any
 
     c_println!("Log Configuration ({})", logs.version());
```

Comment:
Can this be `"Log chain {}", logs.version()`? I'm not sure why this is called "log configuration" throughout restatectl, it certainly isn't configuration.